cluster/ci/config/prow/plugins.yaml

```diff
@@ -300,6 +300,8 @@ config_updater:
       name: cluster-profile-gcp-ha-static
     cluster/test-deploy/openshift-ansible-aws-rhel/*.yaml:
       name: cluster-profile-oa-aws-rhel
+    cluster/test-deploy/aws-atomic/*.yaml:
```

Comment:
These two don't fit the pattern.  Directory name == variable name.  Please follow the existing pattern for configmap names.

Reply:
Updated